api/internal/git/cloner.go

```diff
@@ -22,11 +22,15 @@ func ClonerUsingGitExec(repoSpec *RepoSpec) error {
 	if err = r.run("init"); err != nil {
 		return err
 	}
+	if err = r.run(
+		"remote", "add", "origin", repoSpec.CloneSpec()); err != nil {
+		return err
+	}
 	ref := "HEAD"
 	if repoSpec.Ref != "" {
 		ref = repoSpec.Ref
 	}
-	if err = r.run("fetch", "--depth=1", repoSpec.CloneSpec(), ref); err != nil {
+	if err = r.run("fetch", "--depth=1", "origin", ref); err != nil {
```

Comment:
I'm fine with these changes staying if you want; it's the api/internal/target/kusttarget.go changes specifically that I think need dedicated attention in a new PR. Sorry if that was unclear.

Reply:
Great! Added these 2 back and they fixed the one failing test